### PR TITLE
WFP-2490-Adding queue name to logs

### PR DESCRIPTION
**Pull request by sambailey-moj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/TierCalculationEventListener.kt [hunk 1]
```diff
@@ -25,13 +28,19 @@ class TierCalculationEventListener(
   }
 
   private fun readMessage(wrapper: String?): CalculationEventData {
-    val (message) = objectMapper.readValue(wrapper, Message::class.java)
-    return objectMapper.readValue(message, CalculationEventData::class.java)
+    val message = objectMapper.readValue(wrapper, SQSMessage::class.java)
```
Comment (wood-n) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/TierCalculationEventListener.kt:31]: You could probably still use destructuring here to get the message and the id out e.g. 
`val (message, messageId) = objectMapper.readValue(wrapper, Message::class.java)`

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/TierCalculationEventListener.kt [hunk 2]
```diff
@@ -25,13 +28,19 @@ class TierCalculationEventListener(
...
+    val queueId = hmppsQueueService.findByQueueName("tiercalcqueue")?.id
+    log.info("Received message from {} with messageId:{}", queueId, message?.messageId)
```
Comment (wood-n) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/TierCalculationEventListener.kt]: Using embedded/template expressions in strings in kotlin can make it more readable e.g.
`log.info("Received message from $queueId with messageId: ${message?.messageId}"`

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/TierCalculationEventListener.kt (hunk 1 above)
Comment (sambailey-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/TierCalculationEventListener.kt]: done

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/TierCalculationEventListener.kt (hunk 2 above)
Comment (sambailey-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/TierCalculationEventListener.kt]: done

**Review by wood-n — APPROVED** [review]
LGTM

**Pull request merged**